spec/beaker-puppet/install_utils/foss_utils_spec.rb

```diff
@@ -668,14 +662,35 @@ def logger
       let( :platform ) { Beaker::Platform.new('debian-7-i386') }
 
       it "downloads a deb file, installs, and updates the apt cache." do
-        expect(subject).to receive(:on).with( host, /wget .*/ ).ordered
+        expect(subject).to receive(:on).with( host, /wget .*puppet-release-.*\.deb/ ).ordered
         expect(subject).to receive(:on).with( host, /dpkg .*/ ).ordered
         expect(subject).to receive(:on).with( host, "apt-get update" ).ordered
         subject.install_puppetlabs_release_repo host
       end
 
     end
 
+    describe "when puppet_collection is 'pc1'" do
+      describe "on yum-based platforms" do
+        let( :platform ) { Beaker::Platform.new('el-7-x86_64') }
+
+        it "installs the old style puppetlabs-release-pc1 packages" do
+          expect(host).to receive(:execute).with( /rpm .*puppetlabs-release-pc1.*\.rpm$/ )
```

Comment:
@kevpl could I get some help with this? I'm not great at rspec, but this looks like it should be correct, except I'm getting the following error:

```
  1) ClassMixedWithDSLInstallUtils#install_puppetlabs_release_repo when puppet_collection is 'pc1' on yum-based platforms installs the old style puppetlabs-release-pc1 packages
     Failure/Error:
       host.install_package_with_rpm( remote, '--replacepkgs',
         { :package_proxy => opts[:package_proxy] } )

       #<Unix::Host:0x007fde520f9d70 @logger=#<Double "logger">, @options={:logger=>#<Double "logger">}, @host_hash={:user=>"root", :group=>"root", :pathseparator=>":", :platform=>"el-7-x86_64"}, @name="fak
evm", @apt_needs_update=true, @external_copy_base=nil, @commands=[]> received :execute with unexpected arguments
         expected: (/puppetlabs-release-pc1.*\.rpm$/)
              got: ("rpm --replacepkgs -Uvh http://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm ")
       Diff:
       @@ -1,2 +1,2 @@
       -[/puppetlabs-release-pc1.*\.rpm$/]
       +["rpm --replacepkgs -Uvh http://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm "]
```